src/server/conn.rs

```diff
@@ -95,10 +94,13 @@ impl Conn {
             read_size: 0,
             file_size: 0,
             payload: None,
-            res: VecDeque::new(),
             last_msg_id: 0,
             snap_scheduler: snap_scheduler,
             store_id: store_id,
+            // send buffer can be grown automatically, first using
+            // DEFAULT_SEND_BUFFER_SIZE is ok. Maybe we should need
+            // max size to shrink later.
+            send_buffer: Some(SendBuffer::new(DEFAULT_SEND_BUFFER_SIZE)),
```

Comment:
I don't think taking ownership is necessary because `sock` and `send_buffer` are different fields of `Conn`.